frontend/scss/photographerListView.scss

```diff
@@ -1,9 +1,154 @@
khi7C29+7/tsWVqAi6ugT/QdDr2Ic9fxqsof1kfAPA90lvV6I6Q0XQ0RTsBDbvQeAR24BT0W1xEzrAWmznHu9+jhAqgo6Wk/4C6fHiNvQ9cZ1WkPZXR02FiqCjb+M/ALzjemBp1YbM2FLsfAvvfowcV6Ei6KKDncPrPQC84xB2ER9TrTmzsgy4hjzX9DURmgk6mY6e2YyMXwEnomeEo+lgJ/VtxL+fUgzNBJ0cg97YHRkPY+vXtIebvTQ7G3gQ/35JPVQEnZyJf+e3MfYAlwGryGtW2AGOxXZrSfF4hTbH1agIurgY/85vczwF/Dtpn+m6BFsvuRX/9s45NBN00Ic9A/Pu/AixFbgIOB5b+BtVP3ActmVarmtD2xrZzgSPcvxvT8M+WtfBQ5P3BnA7tg7uVuAR4KBnQuPoYLO844HPdv86zTEfGd/PgX+gveOpEZ4FEOzoybuAWc55RPUm8HvgXmxDzIeAZ4B3e5xHH3aW8lLgk9ji9xWo4EWTXRH0LoBgF8ot2BtAqW4v8CTwBPA09knec8BLwKvALmyAD03y39fpxnTsh2o2MAf4C+wHbCG2BZXeYqchqyLYhgIIcBL2pYTO22jeEDZz3NuNd7AZ4/CA78P6YQDb0GFqN3J6M527/wG+SgZFsC0FEOAM4HJ0oYm0QRZF8EPeCYzwIHZ7dpJ3IiLCJ7B9Ftcx+ccl4bSpAII90N+PfQYlIr6Gi+AN2HfsyWlbAQS4E1ub9FnvRESETwBHY8/ok5sJtrEAgu2h14ctnBURX4tJdCbY1gII7216qZmgiL/hmeB6EiqCbS6AYDPBfcDnaNcba5EcLSaxItj2Agj2pcir2NthFUERX0kVwQgFEOxTr8eBL5DpR9siLZJMEYw2o1qNvY3SZ1ci/n4BfIXef3tem2gFEOyD+w3AXO9ERCR2EYz42dlDwKeB+70TERHWYvsJhvyOP8ozwCO9CfwM24lE+wmK+DoaWyazjmDPBKMWQLAp93XYbfxniHk7L5KKo4G/xJ7RhymCqRSNk4Er0AacIt7WAV8iyDPBVAog2O3wtdhLEhHxE6YIRr4FPtJO4L+BPyft09RE2m4RQW6HUyqAAAewxZlPYVtqfdg3HZFshSiCKd0CH2kBcCWw0jsRkYytw9YJvuOdyGhSmwGOtAt7MXIIWzeY8p9VpK0WYc/lf0ELZ4IpzwBHWo6dN7LEOxGRTN2AvRhp1Uwwl1nRS1gBBLslzuXPLfUYAi7BjgXVd+jlfJwWzwRzshTYjG22qlBMFA/w3nPkBcDzLcgpcqzHjlwVR33A2djSGe8BoWhn7AG+wwe/cVURrB4bRmlXcTATuAxbPuM9KBTtiEPYx/1zGJuKYPXYgGaCrbEcuAP/QaHwjXuAVUzOAuDZFuQcOTQTbJlTgS34DwxFb2Mb8DWKbw+nIlg9NBNsmX7gHGA7/oND0WzsAM6j2gWoIlg9VARbaBB7CP4K/gNEUW/sBL5LfUta5qMiWDVuREWwlaZghVAzwvixAyt8TWybpplg9VARbLEBbOnMVvwHiqJYbAPOxWb1TdJMsHqoCLZcH3YOwl34DxbF+HE38GV6+6ZRRbB6qAgGMfyN8Vv4DxqFxT78dwFSEaweKoKBzMDeJj6M/8DJNR7DntXOnKCvemU+duvt3S6RYyMqguGsBH6KPXD3HkCpx+vApUx+8XKvqQhWDxXBoAaww5quwb4r9R5IqcRb2GmAa2n+pUYdVASrh4pgcIPYBXsV2oChTLyOfZ97GrYsKZp5qAhWjY3E+MGTCfQDq4GLgD9iH997D662xSHsk8SLsbNdUvj110ywemgmmKC5wJnYm8uc3xw+j83yzsJmTCmah4pg1ah1JpjLlvhRdLCL5FjsHJOVwGLS2zHjXeBx4HfY5rR3Ak9jOy+nbh6wiXSLfC/cBPw9NWyvrwLYfoPYLtbLgL/CzjVZBEz3TKqAN7Bi9wi2u/IfgIeAvZ5JOZuHimBVNwFfBN6u8i9RAYypD1vvtrAbH8OeMc3txnR6N2s8iJ3A91w3nsbOZX6iGy93/x55v3nALVi/STmVi6AKYJoGsMXZM7sxvRvTgI9gb1IHu39fP3brPRxDWMF6F7vFeLsbu7HZ3BvAa914tRutOukrkHloJlhVLTNBEfExD5sxe79ciBw3oyUyImHNQ0VQRVAkY/NQEVQRFMnYXFQEVQRFMqYiqCIokjUVQRVBkazNRccvVI1bUBEUCUszQRVBkaxpJqgiKJI1FcF6imDEvSRFBBXBOmITKoIiYc1BRbBqaCYoEpiKYPXYhIqgSFgqgtVjI9DRdlgiMc3BbucWeicS2FdVAEXimoPdzi3wTiSom1QARWLTTLC8XSqAIvGpCJYz1PHOQEQqewH4HHYGi0zeOyqAImlQESzuGRVAkXS8AHwWFcHJul8FUCQtL6EiOFk3eycgIs2YDTyG/4LjtsY+7OhYEUnUbGAL/sWmjXFNhXYVkSA0Exw9VlZpVBGJQ0Xw/aFnfyKZURG0OAAsr9iWIhKQiiBcUrkVRSSs2cAf8S9EHvEiMK16E4pIZLPIbyZ4CFhTR+OJSHyzyGsmeEE9zSYiqcilCP4K6KupzUQkIakXwYfRcz8RGUeqRXAbtleiiMi4UiuCz6KjAkSkgFSK4POo+IlICbOw52beRUwzPxFxEXUmqJmfiNQi2kxQMz8RqdVMYhRBzfxEpBFtnwlq5icijWrrTFAzPxHpibYVQc38RKSn2lIENfMTERczgQdR8RORTHkVQRU/EWmFXhfBbaj4iUiLzKA3RfBBtKuLiLRQ0zPBG9F+fiLSYk3MBA8A3wc6PfxziIiUUudMcBtwfE+zFxGpqOpMcD/wI2BKrxMXEanDDOABihW+Q8D1wGKHfEVEajUDuJuJC98e4DJgqU+aIiLN6Ae+AWwCdmIvNV7HZoeXAqcCU92y6/p/ed+xbhr7Wj4AAAAASUVORK5CYII='/%3E%3C/defs%3E%3C/svg%3E ");
```

Comment:
rather than inlining this image data here, let's stick it in a file and reference it.